tests/validation/tests/v3_api/test_create_ha.py

```diff
@@ -48,6 +53,60 @@
 export_cmd = "export KUBECONFIG=" + kubeconfig_path
 
 
+def test_remove_rancher_ha():
+    resource_prefix = \
+        CATTLE_TEST_URL.split(".qa.rancher.space")[0].split("//")[1]
```

Comment:
CATTLE_TEST_URL is an IP eg https://1.2.3.4

Reply:
Would using RESOURCE_PREFIX instead of CATTLE_TEST_URL help in deleting k3s clusters too using same method?

Reply:
this method is mainly used for deleting all resources coming from the auto-created HA setup.
We can have a separate method and job to handle the deletion of a k3s cluster.